test/tests/postprocessors/variable_inner_product/gold/variable_inner_product.csv

```diff
@@ -0,0 +1,4 @@
+time,f_dot_f,f_dot_g,norm_f
+0,0,0,0
+1,0.4,4.1633363423443e-17,0.63245553203368
```

Comment:
These are not orthonormal Legendre polynomials, just orthogonal so `P_n * P_m = delta_{n,m} 2 / (2*n+1)`.